tools/assets_generate/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/shurcooL/vfsgen"
+)
+
+func main() {
+	var fs http.FileSystem = http.Dir("ui/build")
+	err := vfsgen.Generate(fs, vfsgen.Options{
+		BuildTags:    "ui_server",
```

Comment:
There should be no build tags when it is built on CI to be served for PD (see github action file)

Reply:
Changed it to support passing build tags.